### DISCUSSION TITLE
Votekick public votes: allow players to specify a silent vote

### DISCUSSION THREAD

**Discussion by lokka30** [piqueserver/piqueserver#728] [Feature Requests]
I think it'd be a neat feature if votekicks allowed players to silently vote, regardless of the public vote setting being enabled. E.g. `/y -s` votes 'yes' in a silent manner. Hopefully this encourages more vote participation in servers as some players would rather not be broadcasted as being in favor of kicking someone.

**Comment by NotAFile** [comment]
Are there any cases where this has actually caused issues

**Comment by lokka30** [comment]
> Are there any cases where this has actually caused issues

I saw it suggested somewhere by @feikname (if I recall the user correctly), though it was an offtopic comment so I'm unsure what issue it was posted on.

No clue if the lack of this feature has ever caused issues, never been an issue for myself.

If this issue is too minor then feel free to close it :)
